package.json

```diff
@@ -62,6 +63,7 @@
         "eslint-plugin-cypress": "^3.4.0",
         "eslint-plugin-json": "^4.0.0",
         "eslint-plugin-vue": "^9.27.0",
+        "happy-dom": "^15.7.4",
```

Comment:
what's this for?

Reply:
I did a quick look at [the Nuxt docs](https://nuxt.com/docs/getting-started/testing) and found:

> you can choose between happy-dom and jsdom for a runtime Nuxt environment

Reply:
I found the option to use jsdom. I followed the error messages which was referring to `happy-dom` even though I saw in the docs that it can use `jsdom`. So it has been changed back,